packages/testing/tests/helpers/populateEnvironmentVariables.ts

```diff
@@ -1,7 +1,7 @@
 // Reserved variables
 process.env._X_AMZN_TRACE_ID = '1-abcdef12-3456abcdef123456abcdef12';
 process.env.AWS_LAMBDA_FUNCTION_NAME = 'my-lambda-function';
-process.env.AWS_EXECUTION_ENV = 'nodejs16.x';
+process.env.AWS_EXECUTION_ENV = 'nodejs20.x';
```

Comment:
node18 as default?

Reply:
This is just an environment variable for the unit tests, I decided to jump to 20 since it's the same we run on the dev environment when we run the tests.

Reply:
This environment variable is used only to do two things:
- Know whether we need to append the UA middleware to a client ([here](https://github.com/arnabrahman/aws-lambda-powertools-typescript/blob/8eeb735c8333a5adb649defa47aa5759fbec24ce/packages/commons/src/awsSdkUtils.ts#L4))
- Know whether we need to disable the Tracer ([here](https://github.com/arnabrahman/aws-lambda-powertools-typescript/blob/8eeb735c8333a5adb649defa47aa5759fbec24ce/packages/tracer/src/config/EnvironmentVariablesService.ts#L9))

The variable is used as a proxy for detecting whether we are running within Lambda or elsewhere (i.e. local/CI). In both cases Powertools only cares to know if the variable is present or not, since it's set automatically by Lambda.

For this reason, since we are running Node.js 20x in our dev environment I set it to this version; also so that for when we drop Node.js 18 we have two files less to touch.

However, I can also see an argument for setting it to Node.js 18 for consistency - if we agree on this I'm okay to revert these two files.

Reply:
Thanks for the context, this clears things up! Because there is no special case for node18, like we had for node16, I suggest we can move to node20 as a default. What do you think?

Reply:
Our development environment is already on Node.js 20, which other default do you mean?

Reply:
Talked about this offline & clarified that we are already using Node.js 20 for dev. Resolving.

Reply:
dev, layer and target, mostly for consistency. So we keep layer and target at 18 and dev at 20 as it is now.